install_config/install/deploy_router.adoc

```diff
@@ -269,9 +280,63 @@ $ iptables -A INPUT -p tcp --dport 10443 -j ACCEPT
 ====
 
 
+[[working-with-multiple-routers]]
+=== Working With Multiple Routers
+
+Multiple routers can be grouped to distribute routing load in the cluster and separate
+tenants to different routers.  Each router in the group shardes a set of routes that are
+tagged using labels. Router sharding over the whole cluster uses ROUTE_LABELS.
+Sharding over a namespace (project) uses NAMESPACE_LABELS.
+
+
+Because each router, by default, uses ports 80 (http) and 443 (https) the routers must be
+placed on nodes in the cluster that are not already using those ports, or the ports must
+be changed using the ROUTER_SERVICE_HTTP_PORT and ROUTER_SERVICE_HTTPS_PORT environment
+variables. When namespace labels are used, the service account for the router must have
```

Comment:
You said that already... I think we need to pick one place and just say it clearly there.